pos_access_right/models/pos_config.py

```diff
@@ -57,33 +57,33 @@ class PosConfig(models.Model):
     )
 
     def _compute_group_negative_qty_id(self):
-        for config in self:
+        for _config in self:
```

Comment:
Hi. Why this change ?

Reply:
@legalsylvain to avoide Finding  config from system Odoo and avoid this error ( Loop control variable 'config' not used within the loop body.) If this is intended, started the name with an underscore.